lib/impl/testing_utils.dart

```diff
@@ -0,0 +1,36 @@
+
+/// Return the list of possible test paths. The given [path] must be in the
+/// `lib/` directory.
+List<String> getPossibleTestPaths(String path, { String separator: '/' }) {
```

Comment:
Ah! I understand the restriction now. Yes, required and asserted makes sense to me.